presto-main/src/main/java/com/facebook/presto/operator/aggregation/AggregationUtils.java

```diff
@@ -88,14 +88,14 @@ public static void updateCorrelationState(CorrelationState state, double x, doub
 
     public static double getCorrelation(CorrelationState state)
     {
-        // Math comes from ISO9075-2:2011(E) 10.9 General Rules 7 c x
```

Comment:
It doesn't really matter. If any of the divisors is 0, it will return infinity or nan, which gets turned into null by the aggregation